src/Symfony/Component/Form/Util/FormUtil.php

```diff
@@ -30,8 +30,6 @@ private function __construct()
      * a form and needs to be consistent. PHP keyword `empty` cannot
      * be used as it also considers 0 and "0" to be empty.
      *
-     * @param mixed $data
```

Comment:
let's keep this one

Reply:
fabbot insists that I remove it. I reverted the changes, but it won't pass the test.